blockchain/reactor.go

```diff
@@ -352,7 +358,9 @@ func (bcR *BlockchainReactor) BroadcastStatusRequest() error {
 // Messages
 
 // BlockchainMessage is a generic message for this reactor.
-type BlockchainMessage interface{}
+type BlockchainMessage interface {
+	ValidateBasic() error
```

Comment:
Is it OK for this interface to have `ValidateBasic` method? Or it must be completely generic?

Reply:
No I think it's good to have this method

Reply:
I agree, I think this method being required is a good idea